js/pages/iptt_report/components/report/tableHeader.js

```diff
@@ -155,7 +155,7 @@ const ColumnHeaderRow = inject('rootStore')(
                         /* # Translators: Column header for a target value column */
                         gettext('Target')
                     } />
-                <HeaderCells.HeaderCell
+                <HeaderCells.LopActualHeaderCell
                     styleWidth={110}
```

Comment:
@sanjuroj  - see what you're saying on the manual styles, but in this section (lines 90-180) you can see that because the layout of the report is very specific column to column I'm setting an explicit width (but _only_ the width) for almost all of the columns.  I could do that with either utility classes (`base-column w-130` or whatever) or with BEM classes (`base-column__unit-of-measure-column`) and set them in SCSS, but that seems like a _less_ DRY solution, given it's adding the class structure to pass just the one property from the stylesheet forward.  I don't like setting HTML style attributes manually, but I think it's less cluttered in this application... what do you think?